### DISCUSSION TITLE
Computing pi: machin like formula

### DISCUSSION THREAD

**Discussion by oceanmar** [osveliz/numerical-veliz#9] [Q&A]
![04C71607-19F3-41C5-A18F-BADE1C032563](https://user-images.githubusercontent.com/26480717/194964698-8a0e9ea9-1db1-45c7-a8dd-52fe229c98b5.jpeg)
Hi saw your video about computing. What about this

**Comment by osveliz** [comment]
It looks like that works. Sorry that I didn't respond when you posted this.

**Reply by oceanmar** [reply]
Thanks.

**Reply by oceanmar** [reply]
Actually im trying this formula in python. And it gives me roughly 2 decimal places wrong for example when i check up to 100 roughly 98 decimal correct last no. This is the formula i used deriving from the one i sent u earlier. Its the last formula.
![42AAEAD6-220D-4537-87D8-EE13233E1FD9](https://user-images.githubusercontent.com/26480717/200640539-b4ec2508-044d-4b7b-b782-a7e62a3db2e8.jpeg)

**Reply by osveliz** [reply]
It looks like the 2arctan(1/4) and -2arctan(1/4) will cancel leaving with arctan(1) which will take a long time to converge.

**Reply by oceanmar** [reply]
Actually is yeah it would be like leibniz pi/4, but when i used this in python, to find n decimal places  need n*2 iteration to have n-2 decimals correctly.
This is the alternative for arctan formula im using:
![932F4679-4C04-417D-8026-3128AA2E6126](https://user-images.githubusercontent.com/26480717/201658931-fe932c99-016a-43b2-b553-8ed20863aaa4.jpeg)


does this mean the big O: for this formula is O(n) and by using the equation above in the photo, ee can actually approximate pi by hand.
